pyproject.toml

```diff
@@ -17,6 +17,9 @@ numpy = "2.2.3"
 scipy = "1.15.2"
 scs = "3.2.7.post2"
 picos = "2.6.0"
+sphinx = "4.0"
+sphinx-image-inverter = "0.1.1"
+sphinx-rtd-theme = "1.0.0"
```

Comment:
The version of sphinx you have here likely conflicts with the one under the `tool.poetry.group.docs.dependencies` section.

Also, we shouldn't need `sphinx-rtd-theme ` specified here in the `.toml` file. I would imagine that `sphinx-image-inverter` would be included in the `docs/requirements.txt` and not here, right?